action.yml

```diff
@@ -24,5 +27,5 @@ outputs:
   updated-files:
     description: "Array of paths to updated package.json files"
 runs:
-  using: "node12"
+  using: "node16"
```

Comment:
@defunctzombie since GHA supports node 16 now, you could also update the package.json esbuild script to target node 16

Reply:
https://linear.app/foxglove/issue/FG-2180/update-action-bump-version-to-build-for-node16